### ISSUE TITLE
Core dump in orchagent when assigning router interface to a vlan with untagged mode 

### ISSUE THREAD

**Issue by justin-lu-lyc** [Azure/sonic-buildimage#2684]
**Description**
Core dump in orchagent when assigning router interface to a vlan with untagged mode 

**Steps to reproduce the issue:**
1. Assign IP to Ethernet20 by editing /etc/sonic/config_db.json
2. create vlan 108
    config vlan add 108
3. config vlan member add -u 108 Ethernet20

**Describe the results you received:**
Add router interface to a vlan with untagged mode is illegal, suppose it should reject this configuration.

**Describe the results you expected:**
 Result : orchagent in swss container core dump 
 orchagent (terminated by SIGABRT (core dumped); not expected)

**Additional information you deem important (e.g. issue happens only occasionally):**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```
syslog for this issue
[accton_load_201811_offical_19_run_untag_vlan_in_router_interface.txt](https://github.com/Azure/sonic-buildimage/files/2987300/accton_load_201811_offical_19_run_untag_vlan_in_router_interface.txt)

**Comment by justin-lu-lyc** [comment]
I thought the reason might be in line 241
sonic NOTICE syncd#syncd: :- exit_and_notify: sending switch_shutdown_request notification to OA

and in line 246, core dump happened
Nov  3 17:28:19.596339 sonic INFO swss.sh[2353]: 2016-11-03 17:28:19,595 INFO exited: orchagent (terminated by SIGABRT (core dumped); not expected)

**Comment by xinliu-seattle** [comment]
Add router interface to a vlan  - this is not supported operation. One team is working on the configuration validation. Once it is done, this will behave differently.

**Comment by justin-lu-lyc** [comment]
Agree. We need to do more configuration validation in sonic-utility.
But I think the orchagent should not crash for a little wrong configuration, maybe it just print some error log and skip that setting?

**Comment by xinliu-seattle** [comment]
Right, this behavior of crash is going to be changed once the config validation is done.

**Comment by jipanyang** [comment]
For now, the mitigation to this problem:  https://github.com/Azure/sonic-swss/pull/770

**Comment by justin-lu-lyc** [comment]
> Right, this behavior of crash is going to be changed once the config validation is done.

Will this feature be release in the end of March? I noticed that the configuration validation is listed in Roadmap SONiC.201903.

**Comment by justin-lu-lyc** [comment]
> For now, the mitigation to this problem: [Azure/sonic-swss#770](https://github.com/Azure/sonic-swss/pull/770)

Thanks for your reply. I use similar way to prevent orchagent from crash, but it leads to the mismatch between CONFIG_DB and ASIC_DB. So maybe the best way to resolve this issue is still configuration validation.
